crates/shared/src/server/body.rs

```diff
@@ -25,126 +25,132 @@ pub trait ToBytes {
     /// let bytes: Bytes = Bytes::from("Hello, world!");
     /// let parsed_bytes: Bytes = bytes.to_bytes();
     /// ```
-    fn to_bytes(self) -> Bytes;
+    fn to_bytes(&self) -> Bytes;
```

Comment:
This is breaking for existing applications who may have implemented custom structures